src/API/FeedState.php

```diff
@@ -524,4 +518,17 @@ private function get_global_error_from_workflow( $workflow ) {
 
 		return '';
 	}
+
+	/**
+	 * Helper function used for fetching local feed config file.
+	 *
+	 * This is temporary as we will need to operate on multiple feed files in the future.
+	 *
+	 * @return string
+	 */
+	private function get_feed_url() {
+		$configs = LocalFeedConfigs::get_instance()->get_configurations();
+		$config  = reset( $configs );
+		return $config['feed_url'];
```

Comment:
This seems like a good place to use the null coalescing operator rather than `reset()` as a way to grab the first element, especially since `$configs` could be an empty array.

```suggestion
		return $configs[0]['feed_url'] ?? '';
```

Reply:
Configs will not be an empty array. It always has at least one config.
We also don't know how to fetch the fist element usin `??` because the `$config` array uses country slugs as keys.
Also this code will be altered when we will introduce full multifeed functionality.

Reply:
Ah, I missed that it was keyed with a country slug rather than a numerical index. This makes sense 👍🏻